README.md

```diff
@@ -16,8 +16,8 @@ Make sure that:
 * You have properly configured your local SSH setup (see below)
 * You have logged into `sdr-infra.stanford.edu` and cloned this repository.
 * You have previously `ssh`-ed into all servers.
-  * NOTE: If you are unsure about this, run `bin/sdr check_ssh -e [qa|stage|prod]` and watch the output for any errors!
-* NOTE: if you run `bin/sdr check_cocina`, you may need to ensure that you have the contribsys gem credentials available for google-books to install the sidekiq-pro gem locally (the credential is already on our deploy target VMs).
+  * NOTE: If you are unsure about this, run `sdr check_ssh -e [qa|stage|prod]` and watch the output for any errors!
```

Comment:
Assume the developer has set up their PATH variable so this just works

Reply:
Could you add a note in the README on what the expectations on what should be added to the PATH variable for this to work?

Reply:
On second thought, I think it's likely we devs will miss a random note in a README to twiddle our shell's `PATH` variable since that's not a thing we often touch outside of setting up a new computer. I'll revert that part of this commit. Thanks for the review!